doc/input-output-reference/src/overview/group-airflow.tex

```diff
@@ -1292,31 +1224,31 @@ \subsubsection{Inputs}\label{inputs-5-004}
+This alpha field contains the name of a schedule (ref. Schedule objects) which contains the maximum source zone or space temperature (in Celsius) above which mixing is shutoff as a function of time. The maximum temperature value in the schedule can be -100$^\circ$C and the maximum value can be 100$^\circ$C. This field is an optional field. If this field is not entered, the maximum source temperature control is not applied.
 
 \textbf{Note:} The maximum source temperature when mixing is shutoff must be greater than or equal to the minimum source temperature when mixing is shutoff at any given time. Otherwise, warnings will be issued and the maximum source shutoff temperature will be set to the minimum source shutoff temperature.
 
 \paragraph{Field: Minimum Outdoor Temperature Schedule Name}\label{field-minimum-outdoor-temperature-schedule-name-2}
 
-This alpha field contains the name of a schedule (ref. Schedule objects) which contains the minimum outdoor temperature (in Celsius) below which mixing is shutoff as a function of time. The minimum temperature value in the schedule can be -100°C and the maximum value can be 100°C. This field is an optional field. If this field is not entered, the minimum outdoor temperature control is not applied.
+This alpha field contains the name of a schedule (ref. Schedule objects) which contains the minimum outdoor temperature (in Celsius) below which mixing is shutoff as a function of time. The minimum temperature value in the schedule can be -100$^\circ$C and the maximum value can be 100$^\circ$C. This field is an optional field. If this field is not entered, the minimum outdoor temperature control is not applied.
 
 \paragraph{Field: Maximum Outdoor Temperature Schedule Name}\label{field-maximum-outdoor-temperature-schedule-name-2}
 
-This alpha field contains the name of a schedule (ref. Schedule objects) which contains the maximum outdoor temperature (in Celsius) above which mixing is shutoff as a function of time. The maximum temperature value in the schedule can be -100°C and the maximum value can be 100°C. This field is an optional field. If this field is not entered, the maximum outdoor temperature control is not applied.
+This alpha field contains the name of a schedule (ref. Schedule objects) which contains the maximum outdoor temperature (in Celsius) above which mixing is shutoff as a function of time. The maximum temperature value in the schedule can be -100$^\circ$C and the maximum value can be 100$^\circ$C. This field is an optional field. If this field is not entered, the maximum outdoor temperature control is not applied.
 
 \textbf{Note:} The maximum outdoor temperature when mixing is shutoff must be greater than or equal to the minimum outdoor temperature which mixing is shutoff at any given time. Otherwise, a warning will be issued and the maximum outdoor shutoff temperature will be set to the minimum outdoor shutoff temperature.
 
```

Comment:
Below here are 3 items that should be changed to average. All those with something/s

![image](https://user-images.githubusercontent.com/5778156/189991285-ad480c56-0bf7-4dbc-ab65-ab226e5ef64f.png)

Reply:
And in the ZoneCrossMixing section below this section.

![image](https://user-images.githubusercontent.com/5778156/189991785-56bf68c7-0c66-463e-a901-3228511cd6ea.png)